share/pot/start.sh

```diff
@@ -528,28 +532,43 @@ _js_start()
 		(
 			# shellcheck disable=SC2046
 			eval $( pot info -E -p "$_pname" )
-			"$_confdir/conf/poststart.sh"
+			"$_confdir/poststart.sh"
 		)
 	fi
 
+	sleep 0.5
+	pkill -f -j "$_pname" "^sleep 1234$"
+
 	wait "$_wait_pid"
+	_exit_code=$?
+
+	echo "{ \"ExitCode\": $_exit_code }" > "$_confdir/.last_run_stats"
 
-	if ! _is_pot_running "$_pname" ; then
+	if [ "$_persist" = "NO" ]; then
+		# non-persistent jails always need to die
 		if [ ! -e "${POT_TMP:-/tmp}/pot_stopped_${_pname}" ]; then
 			start-cleanup "$_pname" "${_iface}"
 		fi
 		rm -f "${POT_TMP:-/tmp}/pot_stopped_${_pname}"
-		if [ "$_persist" = "NO" ]; then
-			return 0
-		else
-			return 1
+		# store exit_code of process
```

Comment:
Is this comment a left-over of a previous iteration?
the exit_code is stored 8 lines above...

Reply:
Seems like it, I'll remove it (I think moving it won't help readability, especially since the line doing it is quite verbose).